administrator/components/com_media/resources/scripts/plugins/translate.js

```diff
@@ -2,13 +2,31 @@
  * Translate plugin
  */
 
+import './../../../node_modules/sprintf-js/src/sprintf';
```

Comment:
You can define an alias in the webpack conf (so it's just import xy from sprintf).. but normaly even import sprintf from sprintfs-js should work. ;)